src/lib/layout/index.spec.ts

```diff
@@ -1,11 +1,13 @@
 import { log } from '@nexus/logger'
+import 'jest-extended'
```

Comment:
I tried [following their instructions](https://github.com/jest-community/jest-extended#typescript) but couldn't get it working in our setup. The colocated unit tests makes things more complicated for us I think. Not sure but after 5-10 minutes couldn't get it work 🙈 .

Funnily enough I worked on this area of tsconfig in our system recently.